src/dosemu-preinstallwin31

```diff
@@ -45,28 +309,41 @@ SOURCES = [
             [
                 'https://archive.org/download/SVGA_EXE/SVGA.EXE'
             ]
+    },
+    {
+        'type': 'tool',
+        'name': 'stuffkey',
+        'short_name': 'stuffkey',
+        'urls':
+            [
+                'http://www.retroarchive.org/garbo/pc/keyboard/stfkey40.zip'
+            ]
     }
     ]
 
 SYSLANG = re.findall(r'([a-z][a-z]+)_.+', os.environ['LANG'])[0]
 WINLANG = 'en'
 WINCODEPAGE = '437'
 for source in SOURCES:
-    if source.get('lang') == SYSLANG:
-        WINLANG = source.get('lang')
-        WINCODEPAGE = source.get('codepage')
+   if source.get('type') == 'windows':
+        for winlangentry in source.get('lang'):
+            if winlangentry == SYSLANG:
+                WINLANG = winlangentry
+                WINCODEPAGE = source.get('codepage')
 
 target = '';
 target_dir = '';
 for source in SOURCES:
     if source.get('type') == 'windows':
-        if source.get('lang') != WINLANG:
+        if WINLANG not in source.get('lang'):
```

Comment:
So does this still have a fall-back
to 437 version if lang didn't match?

Reply:
If there's no match in the earlier for loop, then the values set at line 325 and line 326 are not overridden. They will used here to install the US English version.